Sources/SwiftParser/TokenSpecSet.swift

```diff
@@ -405,7 +405,7 @@ enum DeclarationModifier: TokenSpecSet {
     case .internal: return .keyword(.internal)
     case .isolated: return TokenSpec(.isolated, recoveryPrecedence: .declKeyword)
     case .lazy: return .keyword(.lazy)
-    case .mutating: return .keyword(.mutating)
+    case .mutating: return TokenSpec(.mutating, recoveryPrecedence: .declKeyword)
```

Comment:
`mutating` already has a `TokenPrecedence` of `declKeyword`, so I don’t think this change is needed.

https://github.com/apple/swift-syntax/blob/250b431726c7b05309770caaa9989a4166ea6ba7/Sources/SwiftParser/TokenPrecedence.swift#L229-L233